Fixes #2616 - PB <-> Moray agreements

Robbo599 opened the pull request:
Fixes #2616 

# Summary of changes

Added PB <-> Moray agreements

# Screenshots (if necessary)

hsugden commented on Agreements/Airport/PB_Mor.txt:
```diff
@@ -1 +1,3 @@
-COPX:EGPB:*:*:*:*:PBAPP:Moray:11000:*:^Y905
\ No newline at end of file
+COPX:EGPB:*:*:*:*:PBAPP:Moray:12000:*:^Y905
+COPX:EGPM:*:*:*:*:PBAPP:Moray:12000:*:^Y905
+COPX:EGET:*:*:*:*:PBAPP:Moray:12000:*:^Y905
```
I've done some digging in the SRD, which I hadn't managed to get round to before creating the issue, and found that there are a number of standard routes out of here that don't go via the Y905! Therefore, I think it would be best to define these agreements using a FIXAFTER of ADN. I don't think there are any routes that go southbound via the Y905 that _don't_ route via ADN? 

(The reason why it doesn't make sense to use MOCHA or something closer, however, is that there are some routes that route straight to BEKET/PETOX which I think we would still want this agreement to apply to in a VATSIM context).

hsugden commented on Agreements/Airport/Mor_PB.txt:
```diff
@@ -1 +1,3 @@
-COPX:*:*:SUM:EGPB:*:Moray:PBAPP:*:11000:|SUM
\ No newline at end of file
+COPX:*:*:*:EGPB:*:Moray:PBAPP:*:13000:|SUM
+COPX:*:*:*:EGPM:*:Moray:PBAPP:*:13000:|SUM
+COPX:*:*:*:EGET:*:Moray:PBAPP:*:13000:|SUM
```
Same logic as below (I'd read the other comment before this one... I reviewed in a weird order), I think we should define a FIXBEFORE of ADN. I'm not 100% sure whether EuroScope logic works on the basis of ADN being in the route at _any_ point, or explicitly the last fix before entering the next sector in question... so it may be necessary to _also_ define agreements for FIXBEFORE BEKET/PETOX - this might need testing on sweatbox

hsugden commented on .github/CHANGELOG.md:
```diff
@@ -1,3 +1,6 @@
+# Changes from release 2020/06 to 2020/07
+x. Enhancement - Added PB <-> Moray agreements - thanks to @robbo599 (Lee Roberts)
```
```suggestion
x. Procedure Change (2007) - Added PB <-> Moray agreements - thanks to @robbo599 (Lee Roberts)
```

Robbo599 commented:
@hsugden 
I have added that in and it looks to be working. I have also added a few others in to try and catch the other routes in and out. 

![image](https://user-images.githubusercontent.com/42523604/84782969-98d74000-afe0-11ea-9d3a-00f6cd706bc5.png)

![image](https://user-images.githubusercontent.com/42523604/84782987-9d9bf400-afe0-11ea-8e3d-631ffbcfcc66.png)

hsugden commented:
The agreement only applies to traffic via the Y905 though, so I don't think we want any of the extra agreements - we only want what I've commented above

Robbo599 commented:
@hsugden Changed as requested
